### PR TITLE
Revamp TenantConfiguration

### PR DESCRIPTION
**Pull request by louischan-oursky**

### CODE REVIEW

File: pkg/core/config/newtenant.go [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+	return url
+}
+
+// UserConfiguration represents user-editable configuration
+type UserConfiguration struct {
+	APIKey           string                         `json:"api_key" yaml:"api_key" msg:"api_key"`
+	MasterKey        string                         `json:"master_key" yaml:"master_key" msg:"master_key"`
+	URLPrefix        string                         `json:"url_prefix" yaml:"url_prefix" msg:"url_prefix"`
+	CORS             CORSConfiguration              `json:"cors" yaml:"cors" msg:"cors"`
+	Auth             NewAuthConfiguration           `json:"auth" yaml:"auth" msg:"auth"`
```
Comment (kiootic) [inline pkg/core/config/newtenant.go]: Do we remove the `New` prefix? The old configuration seems deleted.

Comment (louischan-oursky) [reply pkg/core/config/newtenant.go]: Yes. we should. Let me remove all `New` prefixes.

**Comment by louischan-oursky** [comment]
@kiootic Remove the `New` prefix both in filenames and types.

**Comment by rickmak** [comment]
@louischan-oursky How can I enter Standalone mode as there is no envconfig directive in the config?

**Comment by rickmak** [comment]
I found the envconfig `STANDALONE` is working as default the loader use env key that is capitalized of the variable name.

**Pull request merged**